flows/definition/localization.go

```diff
@@ -26,8 +26,8 @@ type itemTranslation map[string][]string
 // }
 type languageTranslation map[uuids.UUID]itemTranslation
 
-// GetTextArray returns the requested item translation
-func (t languageTranslation) GetTextArray(uuid uuids.UUID, property string) []string {
+// returns the requested item translation
+func (t languageTranslation) getTextArray(uuid uuids.UUID, property string) []string {
```

Comment:
made some simplifications to `Flow.Localization` stuff so that the API is just `GetItemTranslation` and `SetItemTranslation `